examples/zone-printer/manifests/zoneprinter-deployment.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: zoneprinter
+  labels:
+    app: zoneprinter
+spec:
+  selector:
+    matchLabels:
+      app: zoneprinter
+  template:
+    metadata:
+      labels:
+        app: zoneprinter
+    spec:
+      containers:
+      - name: frontend
+        image: gcr.io/kubemci-2018/zoneprinter:0.1
```

Comment:
We dont really need 2018 here?

Reply:
ah damn it, this should be `google-samples`.